stack/stack.py

```diff
@@ -1,4 +1,5 @@
 class Stack(object):
+    
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,5 +1,5 @@
 class Stack(object):
-    
+
     def __init__(self):
         self._list = []
 
```